app/controllers/api/v2/trends_controller.rb

```diff
@@ -12,20 +13,25 @@ class TrendsController < V2::BaseController
 
       api :GET, "/trends/", N_("List of trends counters")
```

Comment:
Please modify the apipie text to also indicate this api endpoint is deprecated and what is the new one (also for the other methods)

Reply:
I've found and used the `resource_description` do deprecate this, I'd guess `hammer` is able to use that, but I've not checked. In `/apidoc` it looks quite good.
Please note, the docs for this API are only visible if the plugin is not installed, because the plugin override the `Trend` resource in docs.